cmd/vpu_plugin/vpu_plugin.go

```diff
@@ -0,0 +1,185 @@
+// Copyright 2019 Intel Corporation. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/google/gousb"
+	"os"
+	"time"
+
+	"github.com/intel/intel-device-plugins-for-kubernetes/pkg/debug"
+	dpapi "github.com/intel/intel-device-plugins-for-kubernetes/pkg/deviceplugin"
+	pluginapi "k8s.io/kubelet/pkg/apis/deviceplugin/v1beta1"
+)
+
+const (
+	// Movidius MyriadX Vendor ID
+	vendorID = 0x03e7
+	// Device plugin settings.
+	namespace  = "vpu.intel.com"
+	deviceType = "hddl"
+
+	hddlSockPath     = "/var/tmp/hddl_service.sock"
+	hddlServicePath1 = "/var/tmp/hddl_service_ready.mutex"
+	hddlServicePath2 = "/var/tmp/hddl_service_alive.mutex"
+	ionDevNode       = "/dev/ion"
+)
+
+var (
+	isdebug = flag.Int("debug", 0, "debug level (0..3)")
```

Comment:
afaict, we only use the debug levels of 0 or >0 - so, may as well make this 0..1, otherwise it is misleading that higher values gets you more debug ?

Reply:
Will fix it soon.